frontend/package.json

```diff
@@ -141,6 +141,7 @@
     "start": "react-app-rewired start",
     "build": "cross-env CI=true GENERATE_SOURCEMAP=false INLINE_RUNTIME_CHUNK=false react-app-rewired build",
     "test": "tsc && react-app-rewired test --ci --watchAll=false",
+    "test-u": "tsc && react-app-rewired test --ci --watchAll=false -u",
```

Comment:
I'm okay with this but we already have an "update-snapshots" npm task - perhaps update that one instead?

Reply:
Not sure abut the update-snapshots since it looks like it does some extra stuff. I will happily merge them if the other one is not used/useful

Reply:
I don't believe we are using the other for anything else so I'm okay replacing it with your version

Reply:
`--updateSnapshot == -u`